packages/fluentui/react-northstar/src/components/Alert/Alert.tsx

```diff
@@ -27,6 +27,7 @@ import Text, { TextProps } from '../Text/Text';
 
 import ButtonGroup, { ButtonGroupProps } from '../Button/ButtonGroup';
 import AlertDismissAction, { AlertDismissActionProps } from './AlertDismissAction';
+import { AlertBehaviorProps } from '../../../../accessibility/src/behaviors/Alert/alertBehavior';
```

Comment:
```suggestion
import { AlertBehaviorProps } from '@fluentui/accessibility';
```

Reply:
And `export * from './Alert/alertBehavior';` to `packages/fluentui/accessibility/src/behaviors/index.ts` to export type

Reply:
Done